src/kibana/plugins/vis_types/vislib/tile_map.js

```diff
@@ -31,6 +32,40 @@ define(function (require) {
           filter.geo_bounding_box[field] = event.bounds;
 
           pushFilter(filter, false, indexPatternName);
+        },
+        mapZoomEnd: function (event) {
```

Comment:
https://github.com/elastic/kibana/pull/3713#discussion_r29687964

Adding autoPrecision attr to the mapZoomEnd event to check for this.